### PR TITLE
Allow for groups and checks to be skipped via command line parameter

### PR DESCRIPTION
**Pull request by Wicked7000**
This adds the feature specified in the PR #564 

Currently this is how the workflow works:
- User can add --skip followed by any number of IDS 
- For each id before running checks it will see if it belongs to the current group or check.
- If it belongs to the current group all child checks are skipped, if it belongs to the current check only that current check is skipped.
- If skip is specified in the yaml or the command line parameter it will be skipped.

Although the original PR did have a comment specifying that 1.2.* for example could be good I wasn't sure if that was wanted so if any maintainer could let me know what they think of the current implementation and I can go from there, thanks!

### CODE REVIEW

**Review by yoavrotems — APPROVED** [review]
Hey I really like what you done, its really simple and straight forward, I think that its better the have it without the *, and just as you did, 1.2 then it get 1.2 group and if you did, 1.2.1 then it will be 1.2.1 check,
Since each group suppose to follow the same logic (CIS chapters) then It wouldn't make sense to add *, the only use for it will be 1.2.* and it could lead to a wrong use of 1.*.1 for example where un logically all 1.1.1 1.2.1 1.3.1 tests will be skipped. 
Long story short its seems good and thanks for contributing! 👍
